### ISSUE TITLE
[OAKTrust] Browse by Issue Date

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#207]
**URL:** https://oaktrust-dev.library.tamu.edu/browse/dateissued
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:** incorrect ARIA label usage
**Notes:** Filter dropdowns repeat themselves due to ARIA label and alt text i think. ex "Choose the issue year combo box collapsed. Choose year"
